packages/teleport/src/Databases/Databases.story.tsx

```diff
@@ -18,8 +18,28 @@ import React from 'react';
 import Databases from './Databases';
 import { databases } from './fixtures';
 
-export const Loaded = () => <Databases databases={databases} />;
+type PropTypes = Parameters<typeof Databases>[0];
+
+export const Loaded = () => render({ status: 'success' });
+
+export const Loading = () => render({ status: 'processing' });
+
+export const Failed = () =>
+  render({ status: 'failed', statusText: 'Server Error' });
 
 export default {
   title: 'Teleport/Databases',
 };
+
+function render(attemptOptions: Partial<PropTypes['attempt']>) {
+  const props = {
+    databases,
+    attempt: {
+      status: '' as any,
+      statusText: '',
+      ...attemptOptions,
+    },
+  };
+
+  return <Databases {...props} />;
+}
```

Comment:
Because our component is not very complicated and has only two props, we can just pass in props manually:

```
export const Failed = () => <Databases databases={databases} attempt={{ status: 'failed', statusText: 'Server Error' }}/>

...etc
```

We could've also simplified our helper like this (this is in fact how we normally do it):

```
export const Failed = () => <Databases {...props} attempt={{ status: 'failed', statusText: 'Server Error' }}/>

const props = {
  attempt: { 
   status: 'success' as any,
   statusText: ''
 },
 databases,
}
```
(still too much for our simple component though). I know the example in Nodes do it the way you've done it, but sometimes codes can be simplified.

Reply:
Done :+1: